src/main/java/com/chrosciu/bootcamp/tasks/input/InputUtils.java

```diff
@@ -5,11 +5,20 @@
 import reactor.core.publisher.Flux;
 
 import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class InputUtils {
     public static Flux<String> toFlux(InputStream inputStream) {
-        //TODO: Implement
-        return null;
+        Scanner sc = new Scanner(System.in);
```

Comment:
prawie dobrze, ale tu trzeba wykorzystac przekazany parametr `inputStream` a nie hardcodowac `System.in` :)